*** server.properties.orig	2025-05-12 14:13:30.855032598 +0200
--- server.properties	2025-05-12 14:14:58.997059759 +0200
***************
*** 32,44 ****
--- 32,56 ----
  #   EXAMPLE:
  #     listeners = PLAINTEXT://your.host.name:9092
  #listeners=PLAINTEXT://:9092
+ listeners=CLIENT://:9093,CONSUMER://:9099
  
  # Listener name, hostname and port the broker will advertise to clients.
  # If not set, it uses the value for "listeners".
  #advertised.listeners=PLAINTEXT://your.host.name:9092
+ advertised.listeners=CLIENT://:9093,CONSUMER://:9099
+ inter.broker.listener.name=CONSUMER
  
  # Maps listener names to security protocols, the default is for them to be the same. See the config documentation for more details
  #listener.security.protocol.map=PLAINTEXT:PLAINTEXT,SSL:SSL,SASL_PLAINTEXT:SASL_PLAINTEXT,SASL_SSL:SASL_SSL
+ listener.security.protocol.map=CLIENT:SSL,CONSUMER:PLAINTEXT
+ 
+ security.protocol=SSL
+ 
+ ssl.keystore.type=PEM
+ ssl.keystore.location=../../util/ssl/sonar-kafka-keystore.pem
+ ssl.truststore.type=PEM
+ ssl.truststore.location=../../util/ssl/sonar-ca.crt
+ ssl.endpoint.identification.algorithm=
  
  # The number of threads that the server uses for receiving requests from the network and sending responses to the network
  num.network.threads=3
